packages/shared/src/components/cards/PostTLDRSummary.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement } from 'react';
+import { Post } from '../../graphql/posts';
+import { TLDRSummary } from '../utilities';
+import ShowMoreContent from './ShowMoreContent';
+
+interface PostTLDRSummaryProps {
+  post: Post;
+}
+
+export default function PostTLDRSummary({
+  post,
+}: PostTLDRSummaryProps): ReactElement {
+  return (
+    <TLDRSummary>
```

Comment:
This might also be more inline if it's SummaryContainer or something like that?
What do you think?

Reply:
It makes sense 👍